canonical/jaas-dashboard
# WD-2883 - Replace use-query-params

## Done

- Replace use-query-params with the hook from React Router.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Check that you can filter the model list.
- Check that you can filter applications in a model.
- Check that the access, config, actions etc. panels open and close.

## Details

https://warthogs.atlassian.net/browse/WD-2883
